src/storage/cache/rpc/client.rs

```diff
@@ -297,8 +297,32 @@ where
     #[allow(clippy::mut_from_ref)]
     fn get_stream_mut(&self) -> &mut TcpStream {
         // Current implementation is safe because the stream is only accessed by one thread
-        unsafe { &mut *self.stream.get() }
+        unsafe { std::mem::transmute::<*mut TcpStream, &mut TcpStream>(self.stream.get()) }
+    }
+
+    /// Get resp buffer with mutable reference
+    #[allow(clippy::mut_from_ref)]
+    fn get_resp_buf_mut(&self) -> &mut BytesMut {
+        unsafe { std::mem::transmute::<*mut BytesMut, &mut BytesMut>(self.resp_buf.get()) }
+    }
+
+    /// Get req buffer with mutable reference
+    #[allow(clippy::mut_from_ref)]
+    fn get_req_buf_mut(&self) -> &mut BytesMut {
+        unsafe { std::mem::transmute::<*mut BytesMut, &mut BytesMut>(self.req_buf.get()) }
```

Comment:
just   `  unsafe { std::mem::transmute(self.req_buf.get()) }`  will do

Reply:
OK, remove the type annotation.